foundations/javascript_basics/understanding_errors.md

```diff
@@ -164,4 +162,4 @@ The following questions are an opportunity to reflect on key topics in this less
 This section contains helpful links to related content. It isn't required, so consider it supplemental.
 
 - [MDN JavaScript Errors Reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors)
-- Read the W3schools article to find additional [window console object methods](https://www.w3schools.com/jsref/obj_console.asp)
+- Read the W3schools article to find additional [window console object methods](https://www.w3schools.com/jsref/obj_console.asp). Additionally, watch Steve Griffith's brief [video explanation on Chrome Dev Tools Console](https://www.youtube.com/watch?v=TqPrpglfE9c)
```

Comment:
I think the new AR is nice, good find! It'll be best as its own AR item rather than an extension off the previous. 
Also a small nit to keep consistent with the `devtools` spelling.
Would you mind also adding a `.` to the end of the first AR item as well? Since these two will end with one but the first is missing one.

```suggestion
- Read the W3schools article to find additional [window console object methods](https://www.w3schools.com/jsref/obj_console.asp).
- Additionally, watch Steve Griffith's brief [video explanation on the Chrome devtools Console](https://www.youtube.com/watch?v=TqPrpglfE9c).
```

Reply:
Sounds good!